master/internal/api_experiment.go

```diff
@@ -67,18 +67,12 @@ func (a *apiServer) GetExperiment(
 		return nil, err
 	}
 
-	confBytes, err := a.m.db.ExperimentConfigRaw(int(req.ExperimentId))
+	config, err := a.m.db.ExperimentConfig(int(req.ExperimentId))
```

Comment:
what's the purpose of this? it was this way for a reason - all old exp confs can deser into a `map[string]interface{}` but some will fail deser'ing into the current expconf model.

Reply:
aah I didn't realize that I'll revert.